packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -184,3 +184,69 @@ You can also add the Deployment Group to the Pipeline directly:
 // equivalent to the code above:
 deploymentGroup.addToPipeline(deployStage, 'CodeDeploy');
 ```
+
+### Lambda Applications
```

Comment:
I don't think it's worth it to have a separate section on EC2 and Lambda Applications. They're almost exactly the same - seems like a waste of space to me, and the code is duplicated.